docs/online-boutique-asm-manifests/base/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the OnlineBoutique namespace and the ASM_VERSION sidecar injection, you need to replace:
+# ONLINEBOUTIQUE_NAMESPACE in for-namespace/kustomization.yaml
+# and ONLINEBOUTIQUE_NAMESPACE and ASM_VERSION in for-asm-channel/kustomization.yaml
+# before deploying this file.
+# As an example:
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" for-namespace/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g;s/ASM_VERSION/asm-managed-rapid/g" for-asm-channel/kustomization.yaml
```

Comment:
Same as previous comments. I would suggest removing the examples with `sed` from inside the yaml files. I would do that for all the files. 

Also I feel the line `Sample to illustrate how to customize the OnlineBoutique namespace `` does not tell the user much.

Reply:
Done.